File: 1323-maximum-69-number/1323-maximum-69-number.cpp
class Solution {
public:
    int maximum69Number (int num) {
        string numString = to_string(num);
        for(auto &currchar: numString){
            if(currchar == '6'){
                currchar = '9';
                break;
            }
        }
        return stoi(numString);
    }
};

File: 1323-maximum-69-number/1323-maximum-69-number.py
class Solution:
    def maximum69Number (self, num: int) -> int:
        num = list(str(num))
        if '6' not in num:
            return int(''.join(num))
        rep = num.index('6')
        num[rep] = '9'
        return int(''.join(num))